schemas/external/repo/linked-asset-relationship.descriptor.json

```diff
@@ -0,0 +1,7 @@
+{
+  "@id": "https://ns.adobe.com/descriptors/sample-asset-relationship-descriptor",
+  "@type": "xdm:descriptorOneToOne",
```

Comment:
That is not a valid JSON-LD type.  You probably want this to be `xdm:type` instead.